internal/build/buildc.go

```diff
@@ -10,13 +10,17 @@ import (
 var configureTarget = map[string]string{
 	"amd64": "x86_64-pc-linux-gnu",
 	"i686":  "i686-pc-linux-gnu",
+	"arm64": "aarch64-linux-gnu",
 }
 
 func (b *Ctx) buildc(opts *pb.Build, builder *pb.CBuilder, env []string) (newSteps []*pb.BuildStep, newEnv []string, _ error) {
 	// e.g. ncurses needs DESTDIR in the configure step, too, so just export it for all steps.
 	env = append(env, b.substitute("DESTDIR=${DISTRI_DESTDIR}"))
 
-	target := configureTarget[b.Arch]
+	target, ok := configureTarget[b.Arch]
```

Comment:
We also access configureTarget in buildcmake.go and build.go. It might be good to check the validity of `configureTarget[b.Arch]` earlier in the build, and just once.

Reply:
Pulled in earlier into the start of Build instead